command/user.go

```diff
@@ -284,6 +308,24 @@ func randString(n int) string {
 	return string(bs)
 }
 
+func validCredentials(usr, pwd string) (bool, string) {
```

Comment:
Is better if you have 2 different methods to do the validation `validateUsername` and `validatePassword`.